amd_openvx_extensions/CMakeLists.txt

```diff
@@ -63,7 +80,24 @@ if(GPU_SUPPORT AND OpenCL_FOUND AND BUILD_OPENCL)
         message("-- ${Red}WARNING:FFMPEG Not Found -- amd_media module excluded${ColourReset}")
     endif(FFMPEG_FOUND)
 else()
-    message("-- ${Red}WARNING:OpenCL Not Found -- amd_rpp, amd_loomsl, & amd_media modules excluded${ColourReset}")
+    message("-- ${Red}WARNING:GPU support/OpenCL/FFMPEGNot Found -- amd_loomsl, & amd_media modules excluded${ColourReset}")
+endif()
+
+if(GPU_SUPPORT AND "${BACKEND}" STREQUAL "HIP")
```

Comment:
the "${BACKEND}" STREQUAL "HIP" needs to be changed to HIP_FOUND as we already searched for HIP in line 41-56 above.

Reply:
Done